cleanup/cleanDockerImages/cleanDockerImages.properties

```diff
@@ -0,0 +1,4 @@
+dockerRepos = [
+# 		[["repo_name"], number_of_days_to_keep],
+		[["sample-repo"], 30],
```

Comment:
I am wondering whether we should allow here a null value for number_of_days_to_keep to use only the original way of deleting images (based on properties)

Reply:
done :)